package.json

```diff
@@ -5369,7 +5531,10 @@
       {
         "language": "aspnetcorerazor",
         "scopeName": "text.aspnetcorerazor",
-        "path": "./src/razor/syntaxes/aspnetcorerazor.tmLanguage.json"
+        "path": "./src/razor/syntaxes/aspnetcorerazor.tmLanguage.json",
+        "unbalancedBracketScopes": [
```

Comment:
Needed to address this scenario:
![image](https://github.com/dotnet/vscode-csharp/assets/16968319/64dfba18-cb85-4919-8c1e-96ffc715714d)

Otherwise, VS Code will think there is an unclosed bracket and color it red.

Reply:
I'm not sure if this is the only one - you might want to test less than / greather than as well

Reply:
Ah you're right, and I was missing bit shifting operators as well. Thanks!